src/App.js

```diff
@@ -1,113 +1,122 @@
-import React,{useEffect, useState} from 'react';
-import './App.css';
-import Form from './Components/Form';
-import Header from './Components/Header';
-import TodoList from './Components/TodoList';
-import { BrowserRouter, Route, Switch } from "react-router-dom";
-import Login from './Components/Login';
+import React, { useEffect, useState } from "react"
+import "./App.css"
+import Form from "./Components/Form"
+import Header from "./Components/Header"
+import TodoList from "./Components/TodoList"
+import { BrowserRouter, Route, Switch } from "react-router-dom"
+import Login from "./Components/Login"
+import { createMuiTheme, ThemeProvider } from "@material-ui/core"
+import { orange, red } from "@material-ui/core/colors"
+
+const theme = createMuiTheme({
+  palette: {
+    primary: orange,
+    secondary: red,
+  },
+})
 
 function App() {
+  const [inputText, setInputText] = useState("")
 
-  const [inputText, setInputText] = useState("");
+  const [todos, setTodos] = useState([])
 
-  const [todos, setTodos] = useState([]);
-
-  const [status,setStatus] = useState("all")
+  const [status, setStatus] = useState("all")
 
   const [filteredTodos, setFilteredTodos] = useState([])
-   const [inputName, setInputName] = useState("");
-   const [inputpassword, setinputpassword] = useState("");
+  const [inputName, setInputName] = useState("")
+  const [inputpassword, setinputpassword] = useState("")
 
   const [isAdmin, setIsAdmin] = useState(false)
   const [redirect, setRedirect] = useState(false)
-  
```

Comment:
Can we use states in the component itself rather than passing from parent components

Reply:
Done